### ISSUE TITLE
Not able to install on EC2

### ISSUE THREAD

**Issue by farhankhwaja** [foreverjs/forever#953]
When I am trying to install forever on EC2 its failing. Below are the details:

> Node Version: v6.10.1
> NPM Version: 5.4.2

Error Message:
```
npm ERR! Error: failed to fetch from registry: forever
npm ERR!     at /usr/local/lib/node_modules/npm/lib/utils/npm-registry-client/get.js:139:12
npm ERR!     at cb (/usr/local/lib/node_modules/npm/lib/utils/npm-registry-client/request.js:32:9)
npm ERR!     at Request._callback (/usr/local/lib/node_modules/npm/lib/utils/npm-registry-client/request.js:137:18)
npm ERR!     at Request.callback (/usr/local/lib/node_modules/npm/node_modules/request/main.js:109:22)
npm ERR!     at Request.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/request/main.js:198:58)
npm ERR!     at Request.emit (events.js:88:20)
npm ERR!     at ClientRequest.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/request/main.js:356:14)
npm ERR!     at ClientRequest.emit (events.js:67:17)
npm ERR!     at HTTPParser.onIncoming (http.js:1238:11)
npm ERR!     at HTTPParser.onHeadersComplete (http.js:102:31)
npm ERR! You may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>
npm ERR!
npm ERR! System Linux 4.9.38-16.35.amzn1.x86_64
npm ERR! command "node" "/usr/local/bin/npm" "install" "forever" "-g"
npm ERR! cwd /home/ec2-user
npm ERR! node -v v0.6.8
npm ERR! npm -v 1.1.0-2
npm ERR! message failed to fetch from registry: forever
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     /home/ec2-user/npm-debug.log
npm not ok
```
Can anyone help me with this??

**Comment by farhankhwaja** [comment]
Got the issue. Sry to bother

**Comment by AlexChambers** [comment]
How did you fix this problem @farhankhwaja ?

**Comment by farhankhwaja** [comment]
I had two node versions on my EC2 instance, once was the latest and other one very old.
So, I didn't know about a node version already existing on my Machine, so I went ahead and installed NVM. But then got this erorr.

**Resolution:**

Since I knew that I had installed my node  version with NVM, so i first removed all the installed nvm components. To do this just remove the `.nvm` folder. Now run the command `which node`. This helped in finding the path to NODE (if any exists even after uninstalling NVM) on EC2. If you get a path returned, try to remove that node version and install new.

Hope this helps
